exercises/change/change_test.go

```diff
@@ -0,0 +1,68 @@
+package change
+
+import (
+	"reflect"
+	"testing"
+)
+
+const targetTestVersion = 1
+
+func TestTestVersion(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Fatalf("Found testVersion = %v, want %v", testVersion, targetTestVersion)
+	}
+}
+
+func determineExpected(expected interface{}) (valid bool, list []int) {
+	_, isError := expected.(int)
+	if isError {
```

Comment:
In the type assertion general form,
   t, ok := i.(T)
the 2nd variable is a bool.  So, err did not seem idiomatic for a bool, which is the reason for isError.  Make sense ?